Validation errors with filevault-maven-plugin 1.2.2 in asset-share-commons.all-3.8.4.zip

When the asset-share-commons.all-3.8.4.zip package is embedded into a project which is using maven-filevault-plugin v1.2.2, the build is throwing multiple validation errors like: 
[INFO] [ERROR] ValidationViolation: "jackrabbit-docviewparser: Invalid XML found: Given root node name 'cq:design_dialog' (implicitly given via filename) cannot be resolved. The prefix used in the filename must be declared as XML namespace in the child docview XML as well!", filePath=jcr_root/apps/testing-cif-cloud-vendor-packages/application/install/core-cif-components-apps-1.2.0.zip/jcr_root/apps/core/cif/components/commerce/productlist/v1/productlist/_cq_design_dialog/.content.xml, nodePath=/apps/core/cif/components/commerce/productlist/v1/productlist/cq:design_dialog

Other Error examples-
[ERROR] ValidationViolation: "jackrabbit-docviewparser: Invalid XML found: Given root node name 'cq:design_dialog' (implicitly given via filename) cannot be resolved. The prefix used in the filename must be declared as XML namespace in the child docview XML as well!", filePath=jcr_root/apps/asset-share-commons/components/details/pdf/_cq_design_dialog/.content.xml, nodePath=/apps/asset-share-commons/components/details/pdf/cq:design_dialog
[ERROR] ValidationViolation: "jackrabbit-docviewparser: Invalid XML found: Given root node name 'cq:dialog' (implicitly given via filename) cannot be resolved. The prefix used in the filename must be declared as XML namespace in the child docview XML as well!", filePath=jcr_root/apps/asset-share-commons/components/details/pdf/_cq_dialog/.content.xml, nodePath=/apps/asset-share-commons/components/details/pdf/cq:dialog
[ERROR] ValidationViolation: "jackrabbit-docviewparser: Invalid XML found: Given root node name 'cq:dialog' (implicitly given via filename) cannot be resolved. The prefix used in the filename must be declared as XML namespace in the child docview XML as well!", filePath=jcr_root/apps/asset-share-commons/components/details/video/_cq_dialog/.content.xml, nodePath=/apps/asset-share-commons/components/details/video/cq:dialog
[ERROR] ValidationViolation: "jackrabbit-docviewparser: Invalid XML found: Given root node name 'cq:template' (implicitly given via filename) cannot be resolved. The prefix used in the filename must be declared as XML namespace in the child docview XML as well!", filePath=jcr_root/apps/asset-share-commons/components/modals/download/_cq_template/.content.xml, nodePath=/apps/asset-share-commons/components/modals/download/cq:template
[ERROR] ValidationViolation: "jackrabbit-docviewparser: Invalid XML found: Given root node name 'cq:dialog' (implicitly given via filename) cannot be resolved. The prefix used in the filename must be declared as XML namespace in the child docview XML as well!", filePath=jcr_root/apps/asset-share-commons/components/modals/share/_cq_dialog/.content.xml, nodePath=/apps/asset-share-commons/components/modals/share/cq:dialog
[ERROR] ValidationViolation: "jackrabbit-docviewparser: Invalid XML found: Given root node name 'cq:dialog' (implicitly given via filename) cannot be resolved. The prefix used in the filename must be declared as XML namespace in the child docview XML as well!", filePath=jcr_root/apps/asset-share-commons/components/search/common/_cq_dialog/.content.xml, nodePath=/apps/asset-share-commons/components/search/common/cq:dialog
[ERROR] ValidationViolation: "jackrabbit-docviewparser: Invalid XML found: Given root node name 'cq:dialog' (implicitly given via filename) cannot be resolved. The prefix used in the filename must be declared as XML namespace in the child docview XML as well!", filePath=jcr_root/apps/asset-share-commons/components/search/date-range/_cq_dialog/.content.xml, nodePath=/apps/asset-share-commons/components/search/date-range/cq:dialog
[ERROR] ValidationViolation: "jackrabbit-docviewparser: Invalid XML found: Given root node name 'cq:dialog' (implicitly given via filename) cannot be resolved. The prefix used in the filename must be declared as XML namespace in the child docview XML as well!", filePath=jcr_root/apps/asset-share-commons/components/search/freeform-text/_cq_dialog/.content.xml, nodePath=/apps/asset-share-commons/components/search/freeform-text/cq:dialog
[ERROR] ValidationViolation: "jackrabbit-docviewparser: Invalid XML found: Given root node name 'cq:dialog' (implicitly given via filename) cannot be resolved. The prefix used in the filename must be declared as XML namespace in the child docview XML as well!", filePath=jcr_root/apps/asset-share-commons/components/search/path/_cq_dialog/.content.xml, nodePath=/apps/asset-share-commons/components/search/path/cq:dialog
[ERROR] ValidationViolation: "jackrabbit-docviewparser: Invalid XML found: Given root node name 'cq:dialog' (implicitly given via filename) cannot be resolved. The prefix used in the filename must be declared as XML namespace in the child docview XML as well!", filePath=jcr_root/apps/asset-share-commons/components/search/property/_cq_dialog/.content.xml, nodePath=/apps/asset-share-commons/components/search/property/cq:dialog
[ERROR] ValidationViolation: "jackrabbit-docviewparser: Invalid XML found: Given root node name 'cq:dialog' (implicitly given via filename) cannot be resolved. The prefix used in the filename must be declared as XML namespace in the child docview XML as well!", filePath=jcr_root/apps/asset-share-commons/components/search/sort/_cq_dialog/.content.xml, nodePath=/apps/asset-share-commons/components/search/sort/cq:dialog
[ERROR] ValidationViolation: "jackrabbit-docviewparser: Invalid XML found: Given root node name 'cq:dialog' (implicitly given via filename) cannot be resolved. The prefix used in the filename must be declared as XML namespace in the child docview XML as well!", filePath=jcr_root/apps/asset-share-commons/components/search/tags/_cq_dialog/.content.xml, nodePath=/apps/asset-share-commons/components/search/tags/cq:dialog

--- a/ui.apps/src/main/content/jcr_root/apps/asset-share-commons/components/details/pdf/_cq_dialog/.content.xml
+++ b/ui.apps/src/main/content/jcr_root/apps/asset-share-commons/components/details/pdf/_cq_dialog/.content.xml
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0"
           xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
-          xmlns:granite="http://www.adobe.com/jcr/granite/1.0"
+          xmlns:granite="http://www.adobe.com/jcr/granite/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0"
           jcr:primaryType="nt:unstructured"
           jcr:title="PDF"
           sling:resourceType="cq/gui/components/authoring/dialog"

--- a/ui.apps/src/main/content/jcr_root/apps/asset-share-commons/components/details/video/_cq_dialog/.content.xml
+++ b/ui.apps/src/main/content/jcr_root/apps/asset-share-commons/components/details/video/_cq_dialog/.content.xml
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0"
           xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
-          xmlns:granite="http://www.adobe.com/jcr/granite/1.0"
+          xmlns:granite="http://www.adobe.com/jcr/granite/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0"
           jcr:primaryType="nt:unstructured"
           jcr:title="Video"
           sling:resourceType="cq/gui/components/authoring/dialog"

--- a/ui.apps/src/main/content/jcr_root/apps/asset-share-commons/components/modals/download/_cq_template/.content.xml
+++ b/ui.apps/src/main/content/jcr_root/apps/asset-share-commons/components/modals/download/_cq_template/.content.xml
@@ -17,7 +17,7 @@
   ~ limitations under the License.
   -->
 
-<jcr:root xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
+<jcr:root xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0"
           jcr:primaryType="nt:unstructured"
           assetListTitle="Assets"
           cancelButton="Cancel"

--- a/ui.apps/src/main/content/jcr_root/apps/asset-share-commons/components/modals/share/_cq_dialog/.content.xml
+++ b/ui.apps/src/main/content/jcr_root/apps/asset-share-commons/components/modals/share/_cq_dialog/.content.xml
@@ -18,7 +18,7 @@
   -->
 
 <jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
-          xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
+          xmlns:nt="http://www.jcp.org/jcr/nt/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0"
           jcr:primaryType="nt:unstructured"
           jcr:title="E-mail Share Modal"
           sling:resourceType="cq/gui/components/authoring/dialog"

--- a/ui.apps/src/main/content/jcr_root/apps/asset-share-commons/components/search/common/_cq_dialog/.content.xml
+++ b/ui.apps/src/main/content/jcr_root/apps/asset-share-commons/components/search/common/_cq_dialog/.content.xml
@@ -17,7 +17,7 @@
   ~ limitations under the License.
   -->
 
-<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0"
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0"
           xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
           jcr:primaryType="nt:unstructured">
     <tabs jcr:primaryType="nt:unstructured">

--- a/ui.apps/src/main/content/jcr_root/apps/asset-share-commons/components/search/date-range/_cq_dialog/.content.xml
+++ b/ui.apps/src/main/content/jcr_root/apps/asset-share-commons/components/search/date-range/_cq_dialog/.content.xml
@@ -18,7 +18,7 @@
   -->
 
 <jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:granite="http://www.adobe.com/jcr/granite/1.0"
-          xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
+          xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0"
           jcr:primaryType="nt:unstructured"
           jcr:title="Date Range Filter"
           sling:resourceType="cq/gui/components/authoring/dialog"

--- a/ui.apps/src/main/content/jcr_root/apps/asset-share-commons/components/search/freeform-text/_cq_dialog/.content.xml
+++ b/ui.apps/src/main/content/jcr_root/apps/asset-share-commons/components/search/freeform-text/_cq_dialog/.content.xml
@@ -18,7 +18,7 @@
   -->
 
 <jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:granite="http://www.adobe.com/jcr/granite/1.0"
-          xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
+          xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0"
           jcr:primaryType="nt:unstructured"
           jcr:title="Free-form Text Filter"
           sling:resourceType="cq/gui/components/authoring/dialog"

--- a/ui.apps/src/main/content/jcr_root/apps/asset-share-commons/components/search/path/_cq_dialog/.content.xml
+++ b/ui.apps/src/main/content/jcr_root/apps/asset-share-commons/components/search/path/_cq_dialog/.content.xml
@@ -18,7 +18,7 @@
   -->
 
 <jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:granite="http://www.adobe.com/jcr/granite/1.0"
-          xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
+          xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0"
           jcr:primaryType="nt:unstructured"
           jcr:title="Path Filter"
           sling:resourceType="cq/gui/components/authoring/dialog"

--- a/ui.apps/src/main/content/jcr_root/apps/asset-share-commons/components/search/property/_cq_dialog/.content.xml
+++ b/ui.apps/src/main/content/jcr_root/apps/asset-share-commons/components/search/property/_cq_dialog/.content.xml
@@ -18,7 +18,7 @@
   -->
 
 <jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:granite="http://www.adobe.com/jcr/granite/1.0"
-          xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
+          xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0"
           jcr:primaryType="nt:unstructured"
           jcr:title="Property Filter"
           sling:resourceType="cq/gui/components/authoring/dialog"

--- a/ui.apps/src/main/content/jcr_root/apps/asset-share-commons/components/search/sort/_cq_dialog/.content.xml
+++ b/ui.apps/src/main/content/jcr_root/apps/asset-share-commons/components/search/sort/_cq_dialog/.content.xml
@@ -18,7 +18,7 @@
   -->
 
 <jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:granite="http://www.adobe.com/jcr/granite/1.0"
-          xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
+          xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0"
           jcr:primaryType="nt:unstructured"
           jcr:title="Sort"
           sling:resourceType="cq/gui/components/authoring/dialog"

--- a/ui.apps/src/main/content/jcr_root/apps/asset-share-commons/components/search/tags/_cq_dialog/.content.xml
+++ b/ui.apps/src/main/content/jcr_root/apps/asset-share-commons/components/search/tags/_cq_dialog/.content.xml
@@ -18,7 +18,7 @@
   -->
 
 <jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:granite="http://www.adobe.com/jcr/granite/1.0"
-          xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
+          xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0"
           jcr:primaryType="nt:unstructured"
           jcr:title="Tags Filter"
           extraClientlibs="[core.wcm.components.form.options.v1.editor,asset-share-commons.author.dialog]"

